addons/mail/static/src/components/thread_viewer/thread_viewer.xml

```diff
@@ -32,6 +32,7 @@
                         hasCurrentPartnerAvatar="props.hasComposerCurrentPartnerAvatar"
                         hasSendButton="props.hasComposerSendButton"
                         hasThreadTyping="props.hasComposerThreadTyping"
+                        isCompact="(threadViewer.thread.model === 'mail.channel' and threadViewer.thread.mass_mailing) ? false : undefined"
```

Comment:
@alexkuhn since those values are directly computed from thread, shouldn't we let the composer define it instead of passing it as props?

Reply:
Currently just depends on thread's model, but this is not the case if we consider chatter in chat window.